AK/FixedArray.h

```diff
@@ -70,43 +93,37 @@ class FixedArray {
         VERIFY(index < m_size);
         return m_elements[index];
     }
-
     T const& operator[](size_t index) const
     {
         VERIFY(index < m_size);
         return m_elements[index];
     }
 
-    bool contains_slow(T const& value) const
-    {
-        for (size_t i = 0; i < m_size; ++i) {
-            if (m_elements[i] == value)
-                return true;
-        }
-        return false;
-    }
+    using Iterator = AK::SimpleIterator<FixedArray, T>;
+    using ConstIterator = AK::SimpleIterator<FixedArray const, T const>;
 
-    void swap(FixedArray& other)
-    {
-        ::swap(m_elements, other.m_elements);
-        ::swap(m_size, other.m_size);
-    }
-
-    using ConstIterator = SimpleIterator<FixedArray const, T const>;
-    using Iterator = SimpleIterator<FixedArray, T>;
-
-    ConstIterator begin() const { return ConstIterator::begin(*this); }
     Iterator begin() { return Iterator::begin(*this); }
+    ConstIterator begin() const { return ConstIterator::begin(*this); }
```

Comment:
Reordering these is pointless

Reply:
It's consistent to always have the mutable variant before the constant variant. Before my change, this was inconsistent with other functions (such as `operator[]` and `data()`)

Reply:
Then at the very least put it in a separate commit. The same inconsistencies are present in plenty of other files (e.g. AK/vector.h), because we don't have any rule which one comes first.

Reply:
Okay, I will put these into a separate commit.